sklearn/preprocessing/_target_encoder.py

```diff
@@ -110,6 +129,10 @@ class TargetEncoder(OneToOneFeatureMixin, _BaseEncoder):
         Names of features seen during :term:`fit`. Defined only when `X`
         has feature names that are all strings.
 
+    classes_ : ndarray or None
+        If `target_type_` is 'multiclass', holds the label for each class,
+        otherwise `None`.
```

Comment:
I am wondering if it wouldn't make sense to record the class labels also for binary classification for the sake of consistency (as is done in classifiers).

Reply:
It would be more explicit about that is considered the positive class when the class labels are string valued for instance.

Reply:
In a follow-up PR we could also add explicit control for `pos_label` as we do in scorers and visualization tools.

This is probably useless from a predictive performance point of view, but could be useful to make it easier to inspect a fitted pipeline.

Reply:
Done, should I open an issue for `pos_label`?